gravitee-apim-gateway/gravitee-apim-gateway-core/src/main/java/io/gravitee/gateway/jupiter/core/condition/ExpressionLanguageConditionFilter.java

```diff
@@ -46,6 +47,6 @@ public Maybe<T> filter(RequestExecutionContext ctx, T elt) {
             .filter(Boolean::booleanValue)
             .map(aBoolean -> elt)
             .doOnError(throwable -> log.warn("Error parsing condition {}", condition, throwable))
-            .onErrorComplete();
+            .onErrorComplete(throwable -> throwable instanceof ExpressionEvaluationException);
```

Comment:
I was talking about this line:
`.doOnError(throwable -> log.warn("Error parsing condition {}", condition, throwable))`

which logs whatever the type of the exception and says "Error parsing" while it could also an evaluation exception.

Reply:
you comment was on the wrong line and I miss read it, sorry. I'll move the log bellow.